Lib/typing.py

```diff
@@ -323,6 +323,14 @@ def __init__(self, name, doc):
         self._name = name
         self._doc = doc
 
+    @property
+    def __doc__(self):
+        return self._doc
+
+    @__doc__.setter
+    def __doc__(self, value):
+        self._doc = value
```

Comment:
Do we need this to be settable? If possible I would keep it read-only, but it is not very important.